traefik/templates/rbac/clusterrole.yaml

```diff
@@ -28,7 +28,17 @@ rules:
     resources:
       - services
       - endpoints
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
       - secrets
+    {{- if gt (len $.Values.rbac.accessibleSecrets) 0 }}
```

Comment:
Can you explain why a dollar is required?

Reply:
`$.Values.rbac.accessibleSecrets`

Reply:
Nice catch @ivankatliarchuk,  I removed that dollar symbol from `traefik/templates/rbac/clusterrole.yaml`. It is required in `traefik/templates/rbac/role.yaml` because rules are created inside a `range` loop for namespaces and `$` allows to access values from root.
It is not required in for ClusterRole, just something I forgot to cleanup